include/powsybl/iidm/VoltageLevelViews.hpp

```diff
@@ -129,6 +129,8 @@ class NodeBreakerView {
 
     virtual stdcxx::const_range<unsigned long> getNodes() const = 0;
 
+    virtual stdcxx::Reference<Terminal> getOptionalTerminal(unsigned long node) const = 0;
```

Comment:
You have to create two variants of this method, one `const` that returns a `CReference` and one no-const that returns a `Reference`

Reply:
Ok, done